github_prett/part1_03_arrays.md

```diff
@@ -0,0 +1,14 @@
+# 1. the difference between static and dynamic arrays
+static arrays need specifying their size before allocation while dynamic arrays can grow or shrink automatically
+
+# 2. What are run-time complexities of static array?
```

Comment:
put the operations in the question as well. we shouldn't be worried about remembering what operation we need to give the runtime comp for.